### ISSUE TITLE
Improve the panel toggle aria labels

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2376]
The panel toggles in the sidebar:

![screen shot 2017-08-11 at 19 32 40](https://user-images.githubusercontent.com/1682452/29224791-0ef896a6-7ecc-11e7-886e-a1c469def905.png)

are buttons with aria-label and aria-expanded attributes. The current text used for the aria-labels prepends "Open section: " to the panel titles. This could be improved a bit because the combined effect of the text and aria-expanded state is a bit weird. Depending on the state, they are announced as, for example:
`Open section: Excerpt collapsed`
`Open section: Excerpt expanded`

where the second message doesn't make much sense. The Customizer uses "Toggle section:" but, unfortunately, "toggle" is hard to translate in many languages.

Maybe use just "Section: "?

**Comment by joedolson** [comment]
I think that the entire aria label should be removed. I don't think that "Section" is necessary, as the fact that it's a panel that can be collapsed and expanded is inherently communicated by the aria-expanded attribute. Knowing it's a "Section" doesn't give a user any additional information - they would still have to expand the panel to discover what's inside it. 

Arguably, you could use "Settings" to convey that there are settings inside the panel, which might be useful - however, I don't think it's necessary. 

Regardless, I absolutely think that dropping "Open" is worthwhile, if the attribute is kept at all - it's confusing,  since the control is not always used to open the panel.

**Comment by afercia** [comment]
Just noticed the `label` prop is also wrong and actually produces a `label` attribute in the rendered HTML, which is thus invalid:
`<button type="button" aria-expanded="true" label="Open section: Featured image" ...`

**Comment by afercia** [comment]
Just tested removing the aria-labels and I'd say it works pretty well: simpler and more effective.

I'd also consider to expand the text of the "Document" and "Block" buttons at the top of the sidebar:

<img width="320" alt="screen shot 2017-08-29 at 17 22 09" src="https://user-images.githubusercontent.com/1682452/29829021-a3172dc8-8cde-11e7-9738-a1e482bce5fe.png">

adding two aria-labels "Document settings" and "Block settings" can help clarify it's all about settings. After all, users land on the sidebar (or, ideally they should land on it) after they press the "Settings" button, so clarifying that "Document" and "Block" relate to two main settings sections could help.
